neutron/plugins/ml2/drivers/openvswitch/agent/ovs_neutron_agent.py

```diff
@@ -183,7 +185,7 @@ def __init__(self, bridge_classes, integ_br, tun_br, local_ip,
         super(OVSNeutronAgent, self).__init__()
         self.conf = conf or cfg.CONF
 
-        self.fullsync = True
+        self.fullsync = False
```

Comment:
self.fullsync = True    is done on L336 after reporting state of the agent to neutron server.   So.....  don't want the fullsync to be done until after agent has reported into the server.